assets/js/modules/analytics-4/datastore/audiences.test.js

```diff
@@ -163,6 +166,31 @@ describe( 'modules/analytics-4 audiences', () => {
 				);
 			} );
 		} );
+
+		describe( 'syncAvailableAudiences', () => {
```

Comment:
Update: after writing the above I realised we'll want to add a bit of functionality to update the client-side `availableAudiences` setting at the same time, which we'll also want test coverage for.

Reply:
Updated accordingly, please let me know if this looks good now, thanks!

Reply:
Thanks @nfmohit! These are all good additions. However, the actual return value that I mentioned is not being tested, and it would be good to include that for the sake of completeness.

You could roll this into the existing tests (see my other comment), or add separate cases for them.

Reply:
Updated, please let me know if this looks good now, thanks!

Reply:
LGTM, thanks Nahid!